config/default.yml

```
@@ -723,6 +723,15 @@ Rails/SaveBang:
   AllowedReceivers: []
   SafeAutoCorrect: false
 
+Rails/SchemaComment:
+  Description: >-
+    This cop enforces the use of the `comment` option when adding a new table or column
+    to the database during a migration by calling `create_table`, `add_column`,
+    `t.column` or any other add_column alias method such as `t.integer`, `t.string`.
```

Looking at this again, I don't think it needs to be as detailed.
```suggestion
    This cop enforces the use of the `comment` option when adding a new table or column
    to the database during a migration.
```